platform/android/java/lib/src/org/godotengine/godot/Godot.kt

```diff
@@ -669,6 +669,7 @@ class Godot(private val context: Context) {
 			darkMode = newDarkMode
 			GodotLib.onNightModeChanged()
 		}
+		renderView?.inputHandler?.updateCachedRotation()
```

Comment:
Let's create an `onConfigurationChanged` method in `GodotInputHandler` and call that, this way we can keep `updateCachedRotation()` private and not make it part of the public API.